runtime/doc/diagnostic.txt

```diff
@@ -239,7 +239,9 @@ config({opts}, {namespace})                          *vim.diagnostic.config()*
                                  • severity_sort: (default false) Sort
                                    diagnostics by severity. This affects the
                                    order in which signs and virtual text are
-                                   displayed. Options:
+                                   displayed. When true, higher severities are
+                                   displayed before lower severities (e.g.
+                                   ERROR is displayed before WARN). Options:
```

Comment:
is ERROR (defined as `1`) considered "higher" than WARN (defined as `2`) ? Does this also apply to the min/max filters? so `severity={min=ERROR}` _only_ shows things "more severe" (but lower ordinal value)?

Reply:
Yes that's right. The numbering of the enum items is counter-intuitive (IMO) but is uses to preserve ease of compatibility with the LSP diagnostic severity definitions.

A "higher" severity is a "worse" severity, so ERROR is "greater than" WARN even though their numerical values are the opposite.